platform-core-module/src/main/java/com/softicar/platform/core/module/ajax/session/SofticarAjaxSession.java

```diff
@@ -32,9 +32,9 @@ public static Optional<SofticarAjaxSession> getInstance(HttpSession session) {
 		return new AjaxSessionAttributeManager(session).getInstance(SofticarAjaxSession.class);
 	}
 
-	public boolean isSuperUser() {
+	public boolean isCoreModuleAdmin() {
 
-		return user.get().isSuperUser();
+		return user.get().isCoreModuleAdmin();
```

Comment:
the abbreviation `Admin` is arbitrary and shouldn't we use same term as before, that is, `isAdminitrator()`